System/EnemyArchetypes.ini

```diff
@@ -2255,7 +2255,7 @@ PrimaryWeapon=(Weapon="SwatEquipment.SG552MG",Chance=14)
 PrimaryWeapon=(Weapon="SwatEquipment.M1Super90SG",Chance=14)
 PrimaryWeapon=(Weapon="SwatEquipment.Pump2RifleGripSG",Chance=10)
 PrimaryWeapon=(Weapon="SwatEquipment.ColtSMG",Chance=2)
-PrimaryWeapon=(Weapon="SwatEquipment.FNFAL",Chance=4)
+PrimaryWeapon=(Weapon="SwatEquipment.FNFALMG",Chance=4)
 BackupWeapon=(Weapon="SwatEquipment.USP40HG",Chance=33)
```

Comment:
what about this? this weapon doesn't even exist

Reply:
It does in the code.